src/_pylibmcmodule.c

```diff
@@ -2491,9 +2554,14 @@ static void _make_excs(PyObject *module) {
     PylibMCExc_Error = PyErr_NewException(
             "pylibmc.Error", NULL, NULL);
 
+    PylibMCExc_CacheMiss = PyErr_NewException(
```

Comment:
I don't think this should be a root exception, especially if its only use is to signal failure in deserialization?